src/Drivers/Kloudend.php

```diff
@@ -15,7 +15,7 @@ protected function url($ip)
     {
         $token = config('location.kloudend.token', '');
 
-        return "https://ipapi.co/{$ip}/?key={$token}";
+        return "https://ipapi.co/{$ip}/json/" . (!empty($token) ? "?key={$token}" : '');
```

Comment:
See no difference, you're welcome to use the syntax you like.
Thank you for an awesome library!

Reply:
This actually introduced a bug. Because of operator precedence, the `.` operator is stronger than `?`. Therefore, it evaluates to:
```php
return ("https://ipapi.co/{$ip}/json/" . empty($token)) ? '' : "?key={$token}";
```
which is always true.

I'll submit a PR.